cirq-rigetti/cirq_rigetti/service.py

```diff
@@ -0,0 +1,149 @@
+##############################################################################
+# Copyright 2021 The Cirq Developers
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+from typing import cast
+import cirq
+
+from qcs_api_client.client import build_sync_client
+from qcs_api_client.operations.sync import (
+    get_instruction_set_architecture,
+    get_quilt_calibrations,
+    list_quantum_processors,
+)
+from qcs_api_client.models import (
+    InstructionSetArchitecture,
+    GetQuiltCalibrationsResponse,
+    ListQuantumProcessorsResponse,
+)
+from pyquil.api import QuantumComputer
+from cirq_rigetti.sampler import RigettiQCSSampler
+from cirq_rigetti import circuit_transformers as transformers
+from cirq_rigetti import circuit_sweep_executors as executors
+
+
+_default_executor = executors.with_quilc_compilation_and_cirq_parameter_resolution
+
+
+class RigettiQCSService:
+    """A class to perform common cirq tasks through QCS's API."""
```

Comment:
This seems like a pretty important class.  We might want to add more to this docstring.